modules/javafx.swt/src/main/java/javafx/embed/swt/FXCanvas.java

```diff
@@ -98,6 +81,17 @@
 import org.eclipse.swt.widgets.Display;
 import org.eclipse.swt.widgets.Listener;
 import org.eclipse.swt.widgets.Shell;
+import com.sun.glass.ui.Application;
```

Comment:
We were planning to not re-order imports as part of this exercise as it just adds more information to the PR.

Since this PR has less files it is fine.